### PR TITLE
Docs - Add update highlights

### PR DESCRIPTION
**Pull request by pmbrull**
### Describe your changes :

Hi folks, add here any topic related to the upgrade we should highlight in the docs such as new parameters, renames, JSON schema changes... to help users navigate the details of the upgrade a bit better

#
### Type of change :

- [x] Bug fix
- [x] Improvement
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-docs/content/deployment/upgrade/versions/011-to-012.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+# Upgrade from 0.11 to 0.12
+
+Upgrading from 0.11 to 0.12 can be done directly on your instances. This page will list a couple of details that you
+should take into consideration when running the upgrade.
+
+## Highlights
+
+### Data Profiler and Data Quality Tests
+
+On 0.11, the Profiler Workflow handled tow things:
```
Comment (TeddyCr) [inline openmetadata-docs/content/deployment/upgrade/versions/011-to-012.md:15]:
```suggestion
On 0.11, the Profiler Workflow handled two things:
```

Comment (pmbrull) [reply openmetadata-docs/content/deployment/upgrade/versions/011-to-012.md]: Thanks mate

**Review by nahuelverdugo — COMMENTED** [review]
Should we add something about the ingestion CLI which will stop overwriting service connection data?

**Comment by pmbrull** [comment]
> Should we add something about the ingestion CLI which will stop overwriting service connection data?

Oh yes please, let's add that as part of the SM explanations and highlight here

**Comment by nahuelverdugo** [comment]
> Oh yes please, let's add that as part of the SM explanations and highlight here
I will take care of that then :)

**Review by nahuelverdugo — APPROVED** [review]

**Comment by pmbrull** [comment]
Thanks folks. Merging it in. Let's open a new PR if we missed anything else worth mentioning

**Pull request merged**